broker/duplicate_certs.py

```diff
@@ -5,6 +5,7 @@
 from broker.aws import alb, iam_govcloud
 from broker.extensions import config, db
 from broker.models import ALBServiceInstance, DedicatedALBServiceInstance, Certificate
+from broker.tasks.iam import _delete_server_certificate
```

Comment:
N.B. this import feels lazy, but reorganizing code for one ugly (but functional) import seems like overkill